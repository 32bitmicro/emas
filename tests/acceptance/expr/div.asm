.word 1 / 1
.word 1.0 / 1.0
.word 2 / 2
.word 2.0 / 2.0
.word 0 / 1
.word 0.0 / 1.0
.word -1 / 1
.word -1.0 / 1.0
.word -2 / 2
.word -2.0 / 2.0
.word 65535 / 2
.word 65535.0 / 2.0
.word -32768 / 2
.word -32768.0 / 2.0

