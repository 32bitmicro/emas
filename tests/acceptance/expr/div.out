@ 0x0000 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0001 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0002 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0003 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0004 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x0005 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x0006 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x0007 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x0008 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x0009 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x000a : 0x7fff  /  011 111 1 111 111 111  /  32767
@ 0x000b : 0x7fff  /  011 111 1 111 111 111  /  32767
@ 0x000c : 0xc000  /  110 000 0 000 000 000  /  -16384
@ 0x000d : 0xc000  /  110 000 0 000 000 000  /  -16384
